src/components/Landing/WelcomeModule/WelcomeModule.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
+import { GatsbyImage } from 'gatsby-plugin-image';
 import styles from './WelcomeModule.module.scss';
 
-const WelcomeModule = () => (
+const WelcomeModule = ({ landing }) => (
   <div className={styles['welcome__module']}>
-    <img
-      className={styles['welcome__module__image']}
-      src='/workspace.png'
-    ></img>
+    <GatsbyImage
+      image={landing[0].node.childImageSharp.gatsbyImageData}
```

Comment:
What is `landing` is null or `landing[0]` is null ? Check in mother Component if all te data is ok.

Reply:
Also missing `width` and `height` attributes.